### PR TITLE
Add UBI-init images

### PR DESCRIPTION
**Pull request by ogajduse**
This patch adds the [UBI-init](https://catalog.redhat.com/software/containers/ubi8/ubi-init/5c359b97d70cc534b3a378c8) container images.

### CODE REVIEW

**Review by JacobCallahan — CHANGES_REQUESTED** [review]
Minor change, but can you change the name of `00_sshd.sh` to `01_sshd.sh`? I think this hints more at the sequential nature of the scripts and should make it more clear what order they're being executed in.

File: UBI7-init/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
+FROM registry.access.redhat.com/ubi7-init
+MAINTAINER https://github.com/JacobCallahan
```
Comment (jyejare) [inline UBI7-init/Dockerfile:2]: This could be you as well :)

File: UBI7-init/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+LABEL broker_compatible="True"
+
+ENV INIT_ENABLED 1
+ENV HOME /root
+WORKDIR /root
+
+# set root password
+ARG ROOT_PASSWD=change-me
+RUN echo "root:${ROOT_PASSWD:-default}" | chpasswd
```
Comment (jyejare) [inline UBI7-init/Dockerfile:11]: Wouldn't this be a security threat to echo the password in such a way? Especially if the dockerfile started outside of SatQE team ?

**Review by jyejare — COMMENTED** [review]
Looks Good, just non-blocking comment and a security concern.

File: UBI7-init/Dockerfile (hunk 1 above)
Comment (ogajduse) [reply UBI7-init/Dockerfile:2]: Meh, not interested unless this repository is transferred to the SatelliteQE GitHub organization. :smile: But thank you.

File: UBI7-init/Dockerfile (hunk 2 above)
Comment (ogajduse) [reply UBI7-init/Dockerfile:11]: Can you please elaborate on how would a malicious entity get the password?

running `podman image inspect` on the image built with this Dockerfile, I can not see the password being exposed.

```json
{
        "created": "2024-06-27T16:10:00.679006594Z",
        "created_by": "/bin/sh -c #(nop) ARG ROOT_PASSWD",
        "author": "https://github.com/JacobCallahan",
        "comment": "FROM 8611db347597",
        "empty_layer": true
      },
      {
        "created": "2024-06-27T16:10:02.001162264Z",
        "created_by": "|1 ROOT_PASSWD=dog8code /bin/sh -c echo \"root:${ROOT_PASSWD:-default}\" | chpasswd",
        "author": "https://github.com/JacobCallahan",
        "comment": "FROM ff5377ba2422"
      },
```

What changes when the container is run `outside of SatQE team`?

**Comment by ogajduse** [comment]
@JacobCallahan renamed, but consider that we already have `01_promtail_setup.sh` in our internal repository where we also store the build script. We might also need to rename it if we want to ensure the sequence.

**Review by JacobCallahan — APPROVED** [review]
ACK, great addition!

File: UBI7-init/Dockerfile (hunk 2 above)
Comment (jyejare) [reply UBI7-init/Dockerfile:11]: @ogajduse Consider the scenario where the password is a secret set by some `authority` but the image build is being done from another entity in the team. Wouldn't echoing that password likely to leak it ? Also in general, echoing the password is a bad practice at all times. 


I am not the security expert but I raised this as a security awareness person so I would leave the decision to you and @JacobCallahan .

**Pull request merged**